content/source/docs/enterprise/api/run.html.md

```diff
@@ -346,14 +428,66 @@ Parameter | Description
 
 This endpoint represents an action as opposed to a resource. As such, it does not return any object in the response body.
 
 -> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) or [team token](../users-teams-organizations/service-accounts.html#team-service-accounts).
 
+Status  | Response                  | Reason(s)
+--------|---------------------------|----------
+[202][] | none                      | Successfully queued a cancel request.
+[409][] | [JSON API error object][] | Run was not planning or applying, has not been canceled non-forcefully, or the cool-off period has not yet passed.
+
+### Request Body
+
+This POST endpoint allows an optional JSON object with the following properties as a request payload.
+
+Key path  | Type   | Default | Description
+----------|--------|---------|------------
+`comment` | string | `null`  | An optional explanation for why the run was canceled.
+
+### Sample Payload
+
+This payload is optional, so the `curl` command will work without the `--data @payload.json` option too.
+
+```json
+{
+  "comment": "This run was stuck and would never finish."
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/runs/run-DQGdmrWMX8z9yWQB/actions/cancel
+```
+
+## Forcefully cancel a Run
+
+`POST /runs/:run_id/actions/force-cancel`
+
+Parameter | Description
+----------|------------
+`run_id`  | The run ID to cancel
+
+The `force-cancel` action is like [cancel](#cancel-a-run), but ends the run immediately. Once invoked, the run is placed into a `canceled` state, and the running Terraform process is terminated. The workspace is immediately unlocked, allowing further runs to be queued.
+
+This endpoint enforces a prerequisite that a [non-forceful cancel](#cancel-a-run) is performed first. Once a non-forceful cancel has been performed, an additional cool-off period is enforced. The time remaining in the cool-off period can be found using the [run details endpoint](#get-run-details), in the key `force_cancel_delay_seconds`. Once this time period expires, if the run has still not ended, the force-cancel endpoint may be used successfully.
+
+This endpoint represents an action as opposed to a resource. As such, it does not return any object in the response body.
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) or [team token](../users-teams-organizations/service-accounts.html#team-service-accounts).
```

Comment:
Can anyone with write access on the workspace force-cancel, or does it require admin or something?